src/src/Cells.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react';
+
+class Cells extends Component {
+    state = { 
+        alive:"",
+        dead: "",
+     }
+    render() { 
+        return ( 
+            <div>
+            
+            </div>

```

Comment:
Maybe I'm missing how you're working on this section, but how are you rendering the cells here?